addons/stock/models/stock_move.py

```diff
@@ -151,46 +148,116 @@ def _default_group_id(self):
     restrict_partner_id = fields.Many2one('res.partner', 'Owner ', help="Technical field used to depict a restriction on the ownership of quants to consider when marking this move as 'done'")
     route_ids = fields.Many2many('stock.location.route', 'stock_location_route_move', 'move_id', 'route_id', 'Destination route', help="Preferred route to be followed by the procurement order")
     warehouse_id = fields.Many2one('stock.warehouse', 'Warehouse', help="Technical field depicting the warehouse to consider for the route selection on the next procurement (if any).")
+    has_tracking = fields.Selection(related='product_id.tracking', string='Product with Tracking')
+    quantity_done = fields.Float('Quantity Done', compute='_quantity_done_compute', digits=dp.get_precision('Product Unit of Measure'), inverse='_quantity_done_set',
+                                 states={'done': [('readonly', True)]})
+    show_operations = fields.Boolean(related='picking_id.picking_type_id.show_operations')
+    show_details_visible = fields.Boolean('Details Visible', compute='_compute_show_details_visible')
+    show_reserved_availability = fields.Boolean('From Supplier', compute='_compute_show_reserved_availability')
+    picking_code = fields.Selection(related='picking_id.picking_type_id.code', readonly=True)
+    product_type = fields.Selection(related='product_id.type', readonly=True)
+    additional = fields.Boolean("Whether the move was added after the picking's confirmation", default=False)
+    is_editable = fields.Boolean('Is editable when done', compute='_compute_is_editable')
+
+    @api.multi
+    @api.depends('has_tracking', 'pack_operation_ids', 'location_id', 'location_dest_id', 'is_editable')
```

Comment:
Looks like it also depends on `pack_operation_ids.result_package_id` and `location_id.child_ids` and `location_dest_id.child_ids`. 

To check.